momento-sdk/src/intTest/java/momento/sdk/SimpleCacheClientTest.java

```diff
@@ -64,25 +61,30 @@ void teardown() {
 
   @Test
   public void createCacheGetSetDeleteValuesAndDeleteCache() {
-    String cacheName = UUID.randomUUID().toString();
-    String key = UUID.randomUUID().toString();
-    String value = UUID.randomUUID().toString();
+    final String cacheName = UUID.randomUUID().toString();
```

Comment:
As this will be very common, let's extract "randomUUIDString" to a utility method.

Reply:
and that utility method should take a non-optional prefix

Reply:
Done.